lib/oli_web/telemetry.ex

```diff
@@ -9,7 +9,7 @@ defmodule OliWeb.Telemetry do
   def init(_arg) do
     children = [
       {TelemetryMetricsPrometheus,
-       [metrics: metrics(), port: Application.get_env(:oli, :prometheus_port)]},
+       [metrics: metrics() ++ distribution_metrics(), port: Application.get_env(:oli, :prometheus_port)]},
```

Comment:
Sorry but I'm a bit lost on why this would change anything

Reply:
I missed the part that live dashboard calls the `metrics` fun by default. Thanks for the explanation 😄